Application/EdFi.Ods.Api/Security/AuthorizationStrategies/Relationships/RelationshipsAuthorizationContextData.cs

```diff
@@ -28,6 +28,7 @@ private static class PropertyNames
             public const string StaffUSI = "StaffUSI";
             public const string StudentUSI = "StudentUSI";
             public const string ParentUSI = "ParentUSI";
+            public const string ResponsibilitySchoolId = "ResponsibilitySchoolId";
```

Comment:
This class only contains entries for the baseline EdOrg-derived properties. Role named properties are not "new" properties, they are contextual variations of the main ones. There should be no changes to this class.